### PR TITLE
Add docs for white-labeling-logo

### PR DESCRIPTION
**Pull request by ritiksaxena124**
This PR adds the doc for how to white label a logo.

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@ritiksaxena124 Let's discuss this on Websites's call. 
Add this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit), if you would :)

File: content/en/cloud/miscellaneous/_index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Miscellaneous
```
Comment (leecalcote) [inline content/en/cloud/miscellaneous/_index.md:2]: @ritiksaxena124 @iArchitSharma please move this to the existing white-label section.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
"how to white label a logo" - no. 

"how to customize Layer5 Cloud with your own logo" - yes.

**Comment by leecalcote** [comment]
@iArchitSharma ....

**Comment by iArchitSharma** [comment]
@ritiksaxena124 let me know when its complete

File: content/en/cloud/miscellaneous/_index.md (hunk 1 above)
Comment (leecalcote) [reply content/en/cloud/miscellaneous/_index.md:2]: @iArchitSharma please move this to the existing white-label section.

**Comment by Yashsharma1911** [comment]
This needs a rewrite, @iArchitSharma I'm closing this

**Pull request closed without merging**